src/index.html

```diff
@@ -9,6 +9,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav_item">
+            <a href="#apple" class="link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
```

Comment:
add empty line between _multiline_ siblings 
```suggestion
          </li>
          
          <li class="nav_item">
```

Reply:
fixed